beat/theanof.py

```diff
@@ -16,7 +16,7 @@
 from pymc3.model import FreeRV
 
 import theano.tensor as tt
-import theano
+import theano, copy
```

Comment:
please put import copy in a separate line, same line only if they are from the same package